Makefile

```diff
@@ -294,20 +301,30 @@ docs_serve:
 # Test #
 ########
 
-subsystem-run: test subsystem-clean
+subsystem-run: | test-rest-api subsystem-clean enable-kube-api-for-subsystem test-kube-api subsystem-clean
+
+test-rest-api:
+	$(MAKE) _run_test AUTH_TYPE=rhsso
 
-test:
+test-kube-api:
+	$(MAKE) _run_test AUTH_TYPE=none ENABLE_KUBE_API=true FOCUS=kube-api
+
+_run_test:
 	INVENTORY=$(shell $(call get_service,assisted-service) | sed 's/http:\/\///g') \
 		DB_HOST=$(shell $(call get_service,postgres) | sed 's/http:\/\///g' | cut -d ":" -f 1) \
 		DB_PORT=$(shell $(call get_service,postgres) | sed 's/http:\/\///g' | cut -d ":" -f 2) \
 		OCM_HOST=$(shell $(call get_service,wiremock) | sed 's/http:\/\///g') \
 		TEST_TOKEN="$(shell cat $(BUILD_FOLDER)/auth-tokenString)" \
 		TEST_TOKEN_ADMIN="$(shell cat $(BUILD_FOLDER)/auth-tokenAdminString)" \
 		TEST_TOKEN_UNALLOWED="$(shell cat $(BUILD_FOLDER)/auth-tokenUnallowedString)" \
-		AUTH_TYPE="rhsso" \
```

Comment:
Isn't it necessary?

Reply:
it passes from the caller (test/test-kube-api)

Reply:
it defaults to none,  so I think it is
https://github.com/openshift/assisted-service/blob/32089d7350c106f6e3bb9359812338588c455a10/Makefile#L56

Reply:
@RazRegev you're right.